work11/index.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+const div = document.querySelector(".wrap")
+const ul = document.getElementById("js-list");
+const url = "https://jsondata.okiba.me/v1/json/d4ex5210822124352"
+
+function getData() {
+  const result = new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve(
+        async function () {
+          const response = (await fetch(url)).json();
+          return response
+        }()
+      )
+
+    }, 3000);
+  })
+  return result;
+}
+
+async function displayView() {
+  loading();
+
+  try {
+    const arrayData = await getData();
+    createElement(arrayData);
+  } catch (e) {
+    div.textContent = e.message;
+  } finally {
+    hideLoading();
+  }
+
+}
```

Comment:
What we want to do with this function is to pass the retrieved data to `createElement()`.
It would be different to start `loading();` with this function. This is because loading is only done when the data is being retrieved. When this function starts, it has already finished retrieving the data.

Reply:
Thanks for the fix.

I had not learned enough about Object[key].
I will take this opportunity to study it. 

Also, I hope you will update codesandbox with every revision.
Thank you very much.

Reply:
-It means that the load is meaningless because it starts loading at asynchronous execution, right?

commit number
315e11d

Reply:
@443n0511 
Should loading () be written outside the getData function?
In my opinion
1, getData () is called
2. Get a Promise with fetch ()
3. Detect resolve with then () and execute subsequent processing
So I thought it's okay to write loadig () in getData () ...